src/app/modules/main/wallet_section/send/account_item.nim

```diff
@@ -1,44 +1,126 @@
-import strformat
+import NimQml, strformat
 import ../../../shared_models/wallet_account_item
 import ../../../shared_models/token_model
 import ../../../shared_models/currency_amount
 
 export wallet_account_item
 
-type
-  AccountItem* = ref object of WalletAccountItem
-    assets: token_model.Model
+QtObject:
+  type AccountItem* = ref object of QObject
```

Comment:
@Khushboo-dev-cpp is it possible to achieve the same as we're doing for `WalletAccountItem`, to create a `WalletAccountItemObject` and make `AccountItemObject` inherits from it? That concept will free us from some code repetition later in other places.

Reply:
if I change the WalletAccountItem to QObject type it should work, that was affecting many places, but let me try and do it quickly now :D